docs/guides/hedgehog-engine/rangers/Model Imports/ModelIMP.md

```diff
@@ -0,0 +1,40 @@
+---
+description: Model Importing guide for Sonic Frontiers
+---
+# Model Importing
+
+!!! info
+    This guide uses the following tools:
+
+    - [HedgeArcPack](/tools/hedgehog-engine/common/files/#hedgearcpack){ target="_blank"}
+    -Blender
+    -ModelConverter.exe
+    -ModelFBX
+    -.modelImporter add-on for Blender
+
+# Extracting Models
+
+### Creating and Extracting the Pac File
+Firstly, create a Pac file named after the character and proceed to extract the contents of the pack utilizing the HedgeArcPack tool.

```

Comment:
Add a line break after the ending of each chapter of the guide. Helps with readibility when viewing/editing the raw markdown file